source/cloud/azure/aks.md

```diff
@@ -94,57 +94,19 @@ $ az aks nodepool add \
     --name gpunp \
     --node-count 1 \
     --node-vm-size Standard_NC48ads_A100_v4 \
-    --node-taints sku=gpu:NoSchedule \
```

Comment:
I'm totally on board with this change. My only worry is wont there be two node pools now? The default one and the GPU one? So the test pod could be scheduler on either and not work 50% of the time?

Maybe we also need to scale the default node pool down to `0` for the sake of this example?

Reply:
(not super familiar with Azure, so 50% guessing) I think there will be two node pools, but I also think that there would be two node pools if using the `--node-taints sku=gpu:NoSchedule`. The thing that will change is that the nodes in the second node pool (called `gpunp`) will no longer have a taint.

The pod used to have a toleration for that particular taint. A toleration makes it possible for it to schedule on a node that has a matching taint. But it doesn't guarantee it will be scheduled there. For that we'd have to use something like node affinity.

Having said all this, I think the Pod will be scheduled on one of the nodes with a GPU, without having to scale down the other pool. This is because the Pod lists a GPU as a resource it requires and the kubernetes scheduler will look for a node where this resource request can be fulfilled.

Reply:
Yeah you're right! I was thinking about the `gpu:NoSchedule` like a node selector, but it's the inverse. Just stopping the GPU nodes from getting filled up with non-GPU work. The resource requirements will cause it to land on the right node.